build-tools/mono-runtimes/mono-runtimes.props

```diff
@@ -27,6 +27,12 @@
     <!-- LLVM -->
     <_LlvmNeeded Condition=" '$(AndroidSupportedTargetAotAbis)' != '' ">yes</_LlvmNeeded>
     <_LlvmCanBuild64 Condition=" '$(HostBits)' == '64' ">yes</_LlvmCanBuild64>
+    <_LlvmNeededWindows32></_LlvmNeededWindows32>
```

Comment:
It's to reset them to a known state. Properties may be set in the environment or msbuild command line and in this case we don't want that to be possible